### ISSUE TITLE
ValueError: too many values to unpack (expected 9) in extract_repeats

### ISSUE THREAD

**Issue by olgabot** [velocyto-team/velocyto.py#9]
Using the [repeat masking GTF](https://genome.ucsc.edu/cgi-bin/hgTables?hgsid=611454127_NtvlaW6xBSIRYJEBI0iRDEWisITa&clade=mammal&org=Mouse&db=mm10&hgta_group=allTracks&hgta_track=rmsk&hgta_table=0&hgta_regionType=genome&position=chr12%3A56694976-56714605&hgta_outputType=primaryTable&hgta_outFileName=mm10_rmsk.gtf) from the UCSC genome browser, `velocyto extract_repeats` results in the following error:

```
$ velocyto extract_repeats mm10_rmsk.gtf
2017-10-31 17:50:58,709 - DEBUG - Sorting by `sort -k1,1 -k7,7 -k4,4n /home/ubuntu/mm10_rmsk.gtf > /home/ubuntu/mm10_rmsk_sorted.gtf`
2017-10-31 17:51:05,265 - DEBUG - Opening /home/ubuntu/mm10_rmsk_sorted.gtf, output will be at /home/ubuntu/mm10_rmsk_joined.gtf
Traceback (most recent call last):
  File "/home/ubuntu/anaconda/bin/velocyto", line 11, in <module>
    load_entry_point('velocyto==0.9.9', 'console_scripts', 'velocyto')()
  File "/home/ubuntu/anaconda/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/ubuntu/anaconda/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/ubuntu/anaconda/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ubuntu/anaconda/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ubuntu/anaconda/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/ubuntu/anaconda/lib/python3.6/site-packages/velocyto/commands/extract_repeats.py", line 54, in extract_repeats
    chrom, feature_class, feature_type, start, end, junk, strand, junk, tags = fields
ValueError: too many values to unpack (expected 9)
```
 
This looks like a malformed GTF issue or something else to do with reading each line of the gtf.

**Comment by gioelelm** [comment]
It could be a bug, but before start debugging could I ask to check that you correctly selected the output format like below. I got a similar error before and was related to this.

![image](https://user-images.githubusercontent.com/4210138/32242695-c1b7ad40-be73-11e7-854c-8a2c7f3f4af1.png)

**Comment by gioelelm** [comment]
I think this was related with a problem in the docs. The docs linked to an incorrectly precompiled form, that was causing users to download file the UCSC repeat mask table in the wrong format (the suffix was .gtf but the format was something else). fixed by f9a8faa14043f38cee7bb4e081cec8a6c486250b.
